### DISCUSSION TITLE
macOS Big Sur

### DISCUSSION THREAD

**Discussion by danielfalbo** [qutebrowser/qutebrowser#5662] [General] [answered]
Tried latest qtbrowser on the macOS Big Sur public beta (now available at beta.apple.com) and there are some details to adjust:
- command bar background colour seems locked to white
- when hitting 'f' the background of the hints is gone

**Comment by The-Compiler** [comment, marked as answer]
Thanks for giving it a try! Converting this to an issue as it's something I want to look into: https://github.com/qutebrowser/qutebrowser/issues/5663

Let's continue this there.
